packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -238,14 +260,13 @@ export abstract class TelemetryLogger implements ITelemetryLoggerExt {
 						const value =
 							typeof getterOrValue === "function" ? getterOrValue() : getterOrValue;
 						if (value !== undefined) {
-							// @ts-expect-error what
```

Comment:
this was a leftover artifact of the previous PR, #16385, this is just a bit of clean up to remove the expected error, which was spurious, just had to find typing to work around it.